### PR TITLE
Configure services for Payment view

### PR DESCRIPTION
**Pull request by tareq89**
## Description

Link to the GitHub issue https://github.com/opencrvs/opencrvs-core/issues/9195
core pr: https://github.com/opencrvs/opencrvs-core/pull/9296

| Service                                                     | Fee |
|-------------------------------------------------------------|-----|
| Birth registration before 30 days of date of birth          | 5   |
| Birth registration between 30 days and 365 days of date of birth | 7   |
| Birth registration after 365 days of date of birth          | 15  |

## Checklist

- [x] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [x] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: src/form/v2/birth/forms/print-certificate.ts [hunk 1]
```diff
@@ -531,13 +534,128 @@ export const BIRTH_CERTIFICATE_COLLECTOR_FORM = defineActionForm({
       },
       fields: [
         {
-          id: 'collector.collect.payment.data',
+          id: 'collector.collect.payment.data.after365',
```
Comment (cibelius) [inline src/form/v2/birth/forms/print-certificate.ts:537]: Would an id like `collector.collect.payment.data.after365` make more sense? Since the late reg target is configurable.

File: src/form/v2/birth/forms/print-certificate.ts [hunk 2]
```diff
@@ -531,13 +534,128 @@ export const BIRTH_CERTIFICATE_COLLECTOR_FORM = defineActionForm({
...
+            {
+              type: ConditionalType.SHOW,
+              conditional: and(
+                not(
+                  field('child.dob')
+                    .isAfter()
+                    .days(applicationConfig.BIRTH.LATE_REGISTRATION_TARGET)
+                    .inPast()
+                ),
+                field('child.dob').isBefore().now()
```
Comment (cibelius) [inline src/form/v2/birth/forms/print-certificate.ts:554]: Why is the `field('child.dob').isBefore().now()` needed in the conditionals?

**Review by cibelius — APPROVED** [review]

File: src/form/v2/birth/forms/print-certificate.ts (hunk 2 above)
Comment (tareq89) [reply src/form/v2/birth/forms/print-certificate.ts:554]: The date of birth is before today (not in the future)
